### ISSUE TITLE
Display current operation in status bar

### ISSUE THREAD

**Issue by Julow** [panglesd/diffcessible#12]
Currently, we have a region at the top which displays the current operation.

However, this region is using the "diff" format. For instance, Creation of files are displayed like this:
```
--- a/dev/null
+++ b/path/to/new/file
```

It would be better to have a single, more explicit, line:
```
Creation of `path/to/new/file`
```
and similarly for other operations: `rename`, `deletion`, ...

This requires to replace the `string_of_operation` function in `interactive_viewer.ml` into a function that takes a value of type `Patch.operation` and creates the relevant string from the value.

**Comment by Dezzy12** [comment]
Hello @panglesd , please assign me to this

**Comment by Julow** [comment]
Hi @Dezzy12,

Have you been able to setup OCaml and build and run the project ?

**Comment by Dezzy12** [comment]
I successfully built and tested the tool on some personal sample project initialised with git

![Screenshot from 2024-03-05 15-11-11](https://github.com/panglesd/diffcessible/assets/98283956/0c548f00-0aff-4259-986c-637f54e39822)

**Comment by Julow** [comment]
Fantastic ! The issue is assigned to you :)

**Comment by panglesd** [comment]
Fixed by #27
